### ISSUE TITLE
All screenshots should be uptated

### ISSUE THREAD

**Issue by yjacolin** [qgis/QGIS-Documentation#1890]
Ubuntu will stop to support Unity in the next release. We need to update all screenshots in a new Desktop environment.

Several are available:
- KDE
- Gnome
- others

Please could you propose some idea and make a vote.

**Comment by SrNetoChan** [comment]
I think this is a problem that is still far away. So, I don't think it is a 3.0 milestone. Unity will still be in the LTR for a while and we can keep using it.

But of course, we can start talking about it. Another reason for changing is that in ubuntu, dialogs look good but they are a bit big. So a criterium could be that makes screenshoots smaller.

**Comment by havatv** [comment]
Do we need to only have screenshots from Linux platforms?
As long as the user interface contains the same components, we should be able to live with a mixture of Windows, Mac and Linux platform screen shots.  That would open up for more contributions.

**Comment by SrNetoChan** [comment]
I would oppose of using a mix of operating systems in the documentation. IMHO it gives a messy and unprofessional look to the docs, and a added confusion to the users, that's why some years ago I started doing all screenshots.

After that, we eliminated all OS icons in the figures caption. To me it looks much better than some years ago. 

I know than some of you prefer having a mix of OS screenshots as a way to show diversity. I don't think that documentation is a place for that. 

I used Ubuntu, because that was the OS I was using at the time.

Unfortunately I don't think I have time to do it for QGIS 3.4, so it's open to anyone willing to do it. And, of course, I can't I force anyone to keep doing what I did. 

I would not oppose of using any other OS, as long as it's consistent. At least by section. But, any updated screenshot is better than an inexistent or outdated one, so... It's up to whoever is doing the job.
